docs/howto/features/imagebuilding.md

```diff
@@ -0,0 +1,153 @@
+            scipy:
+              display_name: Jupyter SciPy Notebook
+              slug: scipy
+              kubespawner_override:
+                image: jupyter/scipy-notebook:2023-06-26
+```
+
+## The `binderhub-service` chart
+
+We will use the [binderhub-service](https://github.com/2i2c-org/binderhub-service/) Helm chart to run BinderHub, the Python software, as a standalone service to build and push images with [repo2docker](https://github.com/jupyterhub/repo2docker), next to JupyterHub.
+
+### How it works
+
+The `binderhub-service` installation starts a `docker-api` pod on each of the user nodes via the following [DaemonSet definition](https://github.com/2i2c-org/binderhub-service/blob/main/binderhub-service/templates/docker-api/daemonset.yaml).
+
+The `docker-api` pod setups and starts the [dockerd](https://docs.docker.com/engine/reference/commandline/dockerd/) daemon, that will then be accessible via a mounted unix socket, by the `build pods`.
+
+The `build pods` are created as a result of an image build request, and they must run on the same node as the builder pods to make use of the docker daemon. These pods mount a k8s Secret with the docker config file holding the necessary registry credentials.
+
+### How to set it up
+
+1. Setup a registry and safely store the credentials to push to it
+
+  - If the hub will be deployed on a GCP cluster, follow the steps 4 to 8 from the [binderhub-service installation guide](https://github.com/2i2c-org/binderhub-service?tab=readme-ov-file#installation). When you're done, you should have a gcr.io registry setup and the appropriate credentials to push images to it.
+
+  - If the hub will be deployed on another cloud provider than GCP, then you should setup the desired quay.io repository and robot account to push and pull from it. The steps below describes how to create one under the 2i2c quay.io organization, assuming this is where we wish the repositories that will be build by `binderhub-service` to be pushed.
+
+    - Go to https://quay.io/organization/2i2c
+    - Select the 'Robot Accounts' option on the left menu.
+    - Click 'Create Robot account', give it a memorable name (such as `<cluster_name>_<hub-name>_image_builder`) and click 'Create'
+    - In the next screen, don't select any of the existing repositories, as we need this robot account to have enough permissions to push **any** new repository under the organization and permissions to existing repositories is not needed.
+    - Once done, click the name of the robot account again. This will give you its username and password.
+    - Select the 'Team and Membership' option on the left menu.
+    - Click on the 'Options' wheel of the `owners` team, then select 'Manage Team Members'
+    - Type in the name of the robot account that you created, select it from the list and add it to the `owners` team
+
+
+2. Setup the `binderhub-service` config
+
+  ```yaml
+  binderhub-service:
```

Comment:
I think some chunk of this can be simplified per https://github.com/2i2c-org/infrastructure/pull/3733/files#r1508207972